test/tests/functional/pbs_qmgr.py

```diff
@@ -159,3 +208,143 @@ def test_multi_attributes(self):
             self.logger.info("looking for error, %s" % msg)
             self.assertTrue(msg in e.msg[0])
         self.server.expect(NODE, 'queue', op=UNSET, id=self.mom.shortname)
+
+    @tags('multisched')
+    def test_list_sched(self):
+        """
+        Test to verify that qmgr list sched works for multiple schedulers
+        """
+        # Scenario 1: only default sched present
+        # Scenario 2: multiple schedulers present
+
+        
+        # Scenario 1: only default sched present
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True,
+                                runas=ROOT_USER)
+    
+        self.server.manager(MGR_CMD_LIST, SCHED, id="", expect=True,
+                                runas=ROOT_USER)
+        
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="invalide_scname", expect=True,
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+            
+        # Scenario 2: multiple schedulers present
+        # first add 3 scheds
+        self.setup_sc1()
+        self.setup_sc2()
+        self.setup_sc3()
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="", expect=True,
+                                runas=ROOT_USER)
+        
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc3", expect=True,
+                                runas=ROOT_USER)
+            
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1,sc3,default", expect=True,
```

Comment:
I think the format of manager with multiple id's is to pass it a list instead of a comma delimited string